src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/Dashboard_TripSearchScreen.tsx

```diff
@@ -363,6 +367,9 @@ export const Dashboard_TripSearchScreen: React.FC<RootProps> = ({
                   resetTransportModes={filtersState.resetTransportModes}
                 />
               )}
+              {isFlexibleTransportEnabledInRemoteConfig &&
```

Comment:
If I remember correctly, wasn't it said that the city zones info box should have its own feature toggle? Haven't discussed it in quite some time, so not sure if that has been changed.

Reply:
@gorandalum yes is true and is made in the other PR https://github.com/AtB-AS/mittatb-app/pull/3518 about add the toggle, we were considering release without the toggle and this is why is that here.